webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code><a>RTCError</a></code> event
+      when the data transport is closed with an error.</li>
```

Comment:
It looks like this is in the changelog section of the spec; this shouldn't change, but a couple other places should.

Also, I'd recommend "an RTCErrorEvent" instead of "the RTCError event". And we may want to specify what errorDetail value(s) to use.